src/actions/notificationActions.js

```diff
@@ -0,0 +1,82 @@
+import axios from "axios";
```

Comment:
Do we really need this file when we have notifications file in the actions folder? Remove it please.